internal/sdkv2provider/resource_cloudflare_teams_accounts_test.go

```diff
@@ -110,6 +112,10 @@ resource "cloudflare_teams_account" "%[1]s" {
     udp = false
 	root_ca = true
   }
+  connectivity {
```

Comment:
this is breaking the test suite

```
=== RUN   TestAccCloudflareTeamsAccounts_ConfigurationBasic
    resource_cloudflare_teams_accounts_test.go:23: Step 1/1 error: Error running apply: exit status 1
        
        Error: error updating Teams Account proxy settings for account "f037e56e89293a057740de681ac9abbe": invalid account settings request (2024)
        
          with cloudflare_teams_account.famfislmlv,
          on terraform_plugin_test.tf line 12, in resource "cloudflare_teams_account" "famfislmlv":
          12: resource "cloudflare_teams_account" "famfislmlv" {
        
--- FAIL: TestAccCloudflareTeamsAccounts_ConfigurationBasic (4.24s)
FAIL
FAIL	github.com/cloudflare/terraform-provider-cloudflare/internal/sdkv2provider	5.441s
FAIL
make: *** [testacc] Error 1
```

@TylerStanish could you advise whether this is service side or a bad config here?

Reply:
I think this could be because the value provided in `use_zt_virtual_ip` is somehow `null` in the acceptance test. I can repro the API error when passing null for this field

Having a further look

Reply:
It seems the failing acceptance test was before #3321 was merged. With that change being merged, I see the acceptance test for `TestAccCloudflareTeamsAccounts_ConfigurationBasic` [here](https://github.com/cloudflare/terraform-provider-cloudflare/actions/runs/9393480634/job/25870062554#step:7:1168) succeeds

Reply:
nice find, thank you!

Reply:
Thanks @TylerStanish and @jacobbednarz, this PR is now up to date with master.

Reply:
this is now erroring on a different (server side) issue.

```
=== RUN   TestAccCloudflareTeamsAccounts_ConfigurationBasic
    resource_cloudflare_teams_accounts_test.go:23: Step 1/1 error: Error running apply: exit status 1
        
        Error: error updating Teams Account connectivity settings for account "f037e56e89293a057740de681ac9abbe": error unmarshalling the JSON response error body: invalid character '' looking for beginning of value
        
          with cloudflare_teams_account.bkgnqceubx,
          on terraform_plugin_test.tf line 12, in resource "cloudflare_teams_account" "bkgnqceubx":
          12: resource "cloudflare_teams_account" "bkgnqceubx" {
        
--- FAIL: TestAccCloudflareTeamsAccounts_ConfigurationBasic (7.34s)
FAIL
FAIL	github.com/cloudflare/terraform-provider-cloudflare/internal/sdkv2provider	8.359s
FAIL
make: *** [testacc] Error 1
```

will chat to the team about this one.